DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/ethtool/linux/linux_ethtool_impl.py

```diff
@@ -100,7 +110,7 @@ def parse_show(self, command, output, *argv, **kwarg):
             line = RE_SPACES.sub(" ", line).strip().split(":")
             key = line[0].replace(" ", "_").lower()
             val = " ".join(line[1:])
-            ethtool_info[key] = val
+            ethtool_info[key] = val.strip()
```

Comment:
I would also strip the `key` just in case.
```suggestion
            ethtool_info[key.strip()] = val.strip()
```